yocto/dobi.yaml

```diff
@@ -0,0 +1,310 @@
+
+# CPU01-bringup: Generate dynamic kasfile for image naming
+job=cpu01-bringup-gen-image-version:
+  use: image-kas
+  depends:
+    - yocto-envs
+    - yocto-layer-revparse
+  entrypoint: /git/gen-image-version.sh
+    /git
+    /git/yocto/layer-revs
+    cpu01-bringup
+    /git/yocto/.generated-cpu01-bringup-image-naming.yaml
+  mounts:
+    - mount-buildingblock-gitversion-source-dir
+  sources: yocto/layer-revs
+  artifact: /git/yocto/.generated-cpu01-bringup-image-naming.yaml
+  user: "{user.uid}:{user.gid}"
+  env:
+    - "USER={user.name}"
+  #annotations:
+  #  description: "Generate cpu01-bringup image version"
+
+
+# CPU01-bringup: Interactive yocto shell
+# Warning: Image/Mender artifact names will be defaults!
+job=cpu01-bringup-yocto-shell:
+  use: image-kas
+  depends:
+    - yocto-envs
+  command: "shell /repo/kasfile-cpu01-bringup.yaml:kasfile-image-naming.yaml"
+  mounts:
+    - build-dir-yocto
+    - repo-dir-yocto
+    - download-dir-yocto
+    - sstate-dir-yocto
+    - install-dir-yocto
+  working-dir: /work
+  interactive: true
+  env:
+    - USER_ID={user.uid}
+    - GROUP_ID={user.gid}
+    - TERM=xterm-256color
+    - SHELL=/bin/bash
+    - MENDER_SERVER_URL={env.MENDER_SERVER_URL}
+    - MENDER_TENANT_TOKEN={env.MENDER_TENANT_TOKEN}
+    - IMAGE_NAME_SUFFIX={env.IMAGE_NAME_SUFFIX}
+  annotations:
+    description: "interactive yocto shell cpu01-bringup config"
+
+# CPU01-bringup: Build image
+job=cpu01-bringup-build-image:
+  use: image-kas
+  depends:
+    - cpu01-bringup-gen-image-version
+  sources:
+    - yocto/kasfile-cpu01-bringup.yaml
+    - yocto/kasfile-tdxbsp.yaml
+    - yocto/kasfile-mender.yaml
+  artifact:
+    - yocto/install/images/moducop-cpu01
+  command: "build /repo/kasfile-cpu01-bringup.yaml:.generated-cpu01-bringup-image-naming.yaml"
+  mounts:
+    - build-dir-yocto
+    - repo-dir-yocto
+    - download-dir-yocto
+    - sstate-dir-yocto
+    - install-dir-yocto
+  working-dir: /work
+  interactive: true
+  env:
+    - USER_ID={user.uid}
+    - GROUP_ID={user.gid}
+    - TERM=xterm-256color
+    - SHELL=/bin/bash
+    - MENDER_SERVER_URL={env.MENDER_SERVER_URL}
+    - MENDER_TENANT_TOKEN={env.MENDER_TENANT_TOKEN}
+    - IMAGE_NAME_SUFFIX={env.IMAGE_NAME_SUFFIX}
+  annotations:
+    description: "Build cpu01-bringup-image"
+
+
+# CPU01-edgefarm: Generate dynamic kasfile for image naming
+job=cpu01-edgefarm-gen-image-version:
+  use: image-kas
+  depends:
+    - yocto-envs
+    - yocto-layer-revparse
+  entrypoint: /git/gen-image-version.sh
+    /git
+    /git/yocto/layer-revs
+    cpu01-edgefarm
+    /git/yocto/.generated-cpu01-edgefarm-image-naming.yaml
+  mounts:
+    - mount-buildingblock-gitversion-source-dir
+  sources: yocto/layer-revs
+  artifact: /git/yocto/.generated-cpu01-edgefarm-image-naming.yaml
+  user: "{user.uid}:{user.gid}"
+  env:
+    - "USER={user.name}"
+
+
+# CPU01-edgefarm: Interactive yocto shell
+# Warning: Image/Mender artifact names will be defaults!
```

Comment:
To all those warnings: 
They will be defaults unless you override them with the mender.env file.
So please rephrase this a little to make that clear or delete the comment.

Reply:
Rephrased